src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx

```diff
@@ -70,6 +70,20 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
     };
   }
 
+  public async getBountyEth(values: IFormValues): Promise<string> {
+    const currentAction = this.state.currentAction;
+    var ethToSend: number = 0;
+
+    for (var field of currentAction.getFields()) {
+      if (field.name === '_depositAmount' || field.name === '_amount') {
```

Comment:
This is called for all proposals on all schemes (including new schemes we do not know yet), and so it seems that any proposal in a scheme that has fields called `_amount` and/or `_depositAmount` as a field name will add send ether with the call. This may not be appropriate for other schemes - you should guard this calculation buy checking if this is indeed a BountyNetwork proposal

Reply:
I agree. I'll make this change